src/index.js

```diff
@@ -27,9 +29,9 @@ export default class toColor {
   getColor(count = 0) {
     const h = this._pickHue();
     const s = this._pickSaturation(h);
-    const b = this._pickBrightness(h, s);
-    const hsl = this._HSVtoHSL(h, s, b);
-    const formatted = this._formatHSL(hsl);
+    const l = this._pickLightness(h, s);
```

Comment:
This doesn't use inputs either, right?

```suggestion
    const l = this._pickLightness();
```